kano_updater/ui/install_window.py

```diff
@@ -54,6 +62,29 @@ def __init__(self):
 
         self._start_install()
 
+    def _launch_game(self, window=None, event=None):
+        try:
+            if event and event.get_keycode()[1] == 44:  # [J] key
```

Comment:
You can use a Gdk constant here instead: `Gdk.KEY_j`

https://lazka.github.io/pgi-docs/index.html#Gdk-3.0/constants.html#Gdk.KEY_j